Sources/GRPCCore/Call/Client/ClientInterceptor.swift

```diff
@@ -102,7 +102,7 @@ public protocol ClientInterceptor: Sendable {
   func intercept<Input: Sendable, Output: Sendable>(
     request: ClientRequest.Stream<Input>,
     context: ClientInterceptorContext,
-    next: @Sendable (
+    next: (
```

Comment:
you might want to adopt `sending` here instead. So that you can call an interceptor on a child task if one wants to

Reply:
Yeah I was thinking about that. I was going to go through this module separately and adopt `sending` since we're still using v5 language mode and I think we need it in a few places.